Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -396,6 +398,11 @@ class LocalDOMWindow final
     void failedToRegisterDeviceMotionEventListener();
 #endif
 
+    InteractionID computeInteractionID(const Event&, EventType);
+    InteractionID& ensureUserInteractionValue();
```

Comment:
This should be a private function now that it's only called by other member functions.